unstable/doc.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2016-2023, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// A package that exposes an unstable interface across go module boundaries. This is
+// necessary to get around granularity limitations of the `internal` path that go modules
+// provides.
+//
+// It is not recommended to rely on exports from this package. We do not provide stability
+// guarantees for this package.
+package unstable
```

Comment:
Should this just go to x or unstable is a meaningful distinction?

Reply:
There is a distinction, though I'm not sure it's meaningful. I would like the functionality in `x` to mature and be stabilized. Ideally, we will depreciate `x` for the `tfbridge` and we become confident in these APIs. `unstable` is an implementation detail of the bridge, and should never be depended upon out of repo. We don't want to stabilize it.